apps/balancer-tools/src/app/apr/pool/(components)/HistoricalCharts/HistoricalData/HistoricalSwapFeeChart.tsx

```diff
@@ -0,0 +1,46 @@
+import { amberDarkA, blueDarkA } from "@radix-ui/colors";
+import { PlotType } from "plotly.js";
+
+import { PoolStatsResults } from "#/app/apr/api/route";
+
+import { generateAndTrimAprCords, getRoundName } from "..";
+
+export default function HistoricalSwapFeeChartData(
+  apiResult: PoolStatsResults,
+  roundId?: string,
+): Plotly.Data {
+  const normalBarColor = blueDarkA.blueA9;
+  const highlightedBarColor = amberDarkA.amberA9;
+
+  const trimmedCollectedSwapFeeData = generateAndTrimAprCords(
+    apiResult.perRound,
+    (result) => result.collectedFeesUSD,
+    0,
+  );
+
+  const barColors = new Array(trimmedCollectedSwapFeeData.x.length).fill(
+    normalBarColor,
+  );
+
+  const colletedSwapFeePerRoundData = {
+    name: "Collected SwapFee",
+    hovertemplate: "Weekly Fees: $%{y:.2f}<extra></extra>",
+    x: trimmedCollectedSwapFeeData.x,
+    y: trimmedCollectedSwapFeeData.y,
+    marker: {
+      color: barColors,
+    },
+    line: { shape: "spline" } as const,
```

Comment:
would do bar for everything that's not APR -- and also add new vertical axis so that all values are still readable